lib/redis/store/namespace.rb

```diff
@@ -48,7 +48,7 @@ def mget(*keys)
         if keys.any?
           # Marshalling gets extended before Namespace does, so we need to pass options further
           if singleton_class.ancestors.include? Marshalling
-            super(*keys.map {|key| interpolate(key) }, options)
+            super(*keys.map {|key| interpolate(key) }, options || Hash.new)
```

Comment:
I would prefer that, rather than do the check here right when callign super, if you changed the assignment of `options` on line 47 to:

```ruby
options = (keys.pop if keys.last.is_a? Hash) || {}
```

This is just so that down the road, this method will be a little easier to read in case a bug occurs somewhere in the code.